### DISCUSSION TITLE
Error during install Debian 10 - buster

### DISCUSSION THREAD

**Discussion by mjswiatk** [GerryDazoo/Slinger#489] [General]
Here are my errors during install on Debian 10 - buster.    Any idea why line 367 is invalid?

sling.log
SyntaxError: invalid syntax
  File "slingbox_server.py", line 367
    nonlocal sid, seq, s_ctl, dbuf, skey, stat, smode

sysop@slingy:~$ sudo systemctl status sling
● sling.service - SlingBox Server Service
   Loaded: loaded (/etc/systemd/system/sling.service; enabled; vendor preset: enabled)
   Active: failed (Result: exit-code) since Thu 2023-01-12 11:08:09 CST; 7s ago
  Process: 1278 ExecStart=/usr/bin/env python -u slingbox_server.py (code=exited, status=1/FAILURE)
 Main PID: 1278 (code=exited, status=1/FAILURE)

Jan 12 11:08:09 slingy systemd[1]: sling.service: Service RestartSec=100ms expired, scheduling restart.
Jan 12 11:08:09 slingy systemd[1]: sling.service: Scheduled restart job, restart counter is at 5.
Jan 12 11:08:09 slingy systemd[1]: Stopped SlingBox Server Service.
Jan 12 11:08:09 slingy systemd[1]: sling.service: Start request repeated too quickly.
Jan 12 11:08:09 slingy systemd[1]: sling.service: Failed with result 'exit-code'.
Jan 12 11:08:09 slingy systemd[1]: Failed to start SlingBox Server Service.

**Comment by GerryDazoo** [comment]
Your using a 2.7 version of python. You must have V3.5 or later.

**Comment by mjswiatk** [comment]
Thanks!  I just edited the service file to be python3.7 instead of just
python.  Works great!!
